### ISSUE TITLE
Slowness After Uploading SVG SET

### ISSUE THREAD

**Issue by SekSion** [soflyy/oxygen-bugs-and-features#1613]
**Describe the bug**

So i needed some icons from font-awesome that oxygen does not come from Default, so i went into IcoMoon and upladed Every icon from Font-awesome 5.15, (I think oxygen should always update font-awesome when there are updates).

Therefore i created the svg set and uploaded into oxygen using Oxygen Settings.

The thing is after that the whole site is slow, it seems to slow at least 20 seconds or more.

*A clear and concise description of what the bug is.*

Uploading a new SVG SET will slow down the whole site (backoffice included).

**Have you tried all the steps at https://oxygenbuilder.com/documentation/troubleshooting/troubleshooting-guide/?**
Yes

**Are you able to replicate the issue on a Sandbox install at https://oxygenbuilder.com/try?**
*If yes, provide the link to a Sandbox install where the issue is present. If the issue only exists on a specific post or template, provide a direct link to that post or template.*

I can and cannot, so the sandbox does not stay slow, but the upload of svg set is not working, so when i try to use the svg set on the sandbox it doenst show any icons at all.

EDIT: Using the 5.13 from the url below, it does work on sandbox and there is no slowness...
http://practical-finch.w5.wpsandbox.pro?upass=xnEV50yKa4J6

While on my website, it does work and i can search the icons but i have the slowness everywhere. Even regenerating CSS Cache takes more than 2 minutes for 8 templates.

**Expected behavior**
*A clear and concise description of what you expected to happen.*
I would expect for the icons to show up  using the icon component and the speed or performance of the website not affected.

**What actually happens**
*A clear and concise description of what happens instead of the expected behavior.*
Website becomes slow, like really slow

**Desktop (please complete the following information):**
- OS: Windows 10
 - Browser: Microsoft Edge, Tested also with brave and chrome all do the same.
 - Oxygen Version 3.5

----

I tried to use this https://oxygen4fun.supadezign.com/resources/what-you-need-to-know-about-font-awesome-in-oxygen/

And also tried a custom icon sets using icomoon.

EDIT: Further investigation i found that its something with admin-ajax.php, its taking 11.32 seconds(sometimes over 20s) to load after uploading a SVG SET, And before uploading a SVG SET it takes less than 500ms to load...

Here is the list of plugins i am using.

Advanced Custom Fields
Custom Post Type UI
Google Analytics for WordPress by MonsterInsights
Gravity Forms
Insert Headers and Footers
Oxygen
Oxygen Elements for WooCommerce
Oxygen Gutenberg Integration
Password Protect WordPress Lite
Post Types Order
Rank Math SEO
Really Simple SSL
reSmush.it Image Optimizer
White Label CMS
WP Cerber Security, Anti-spam & Malware Scan

As for MU-PLUGIN

Force Strong Passwords - WPE Edition 
WP Engine Seamless Login Plugin
WP Engine Security Auditor
WP Engine System

**Comment by KittenCodes** [comment]
@SekSion Please can you send me a link to your SVG defs file?

**Comment by SekSion** [comment]
Yes Give me a moment to provide it

> @SekSion Please can you send me a link to your SVG defs file so that I can download it?

**Comment by SekSion** [comment]
Here is for the whole Zip that icoMoon generates: https://easyupload.io/m0owq4

Here is just the symbol-defs.svg : https://easyupload.io/qgvn7q

**Comment by SekSion** [comment]
I already Tried to think that some other plugin was messing it up but i can't seem to find, it feels like whenever i upload a svg set it's constantly looking for something on every request to admin-ajax.php,

I also thought it could be the size of the svg set but i tried with only 5 icons and it does the same

**Comment by KittenCodes** [comment]
@SekSion Great, thank you. I'll try to replicate the issue so that I can report it internally.

**Comment by KittenCodes** [comment]
@SekSion Unfortunately, the icon titles aren't included in the symbol-defs.svg file, so Oxygen isn't able to read it. Would you be able to include the titles please? Here's how: https://www.loom.com/share/ae1c88e4cec34c8fa5cec6a4cc47de06.

**Comment by SekSion** [comment]
dam, was it really just that? god, let me reply you back with how it went

**Comment by SekSion** [comment]
@KittenCodes  Sorry to say it wasn't that, i did the same as the video and still gets the slowness...

**Comment by SekSion** [comment]
![image](https://user-images.githubusercontent.com/46030872/96266760-eb406180-0fbe-11eb-8af1-acc4adb7e3d9.png)
I even happened to get a 502 cause it took so long

**Comment by SekSion** [comment]
and this is whats holding it .

![image](https://user-images.githubusercontent.com/46030872/96266852-0d39e400-0fbf-11eb-9e03-9a8c29a2ca7e.png)

**Comment by SekSion** [comment]
And AFter it ends this happens. 

![image](https://user-images.githubusercontent.com/46030872/96266993-38243800-0fbf-11eb-9248-ee84868b4417.png)

**Comment by SekSion** [comment]
Just to get back at where i got the icons it was from here: https://fontawesome.com/download

I downloaded Free Web and then uploaded into IconMoon these files from these folders

![image](https://user-images.githubusercontent.com/46030872/96267281-86d1d200-0fbf-11eb-8de1-5747d7ecf784.png)

**Comment by KittenCodes** [comment]
@SekSion I've added the full free Font Awesome icon set to the Sandbox site here: https://oxygen-q8758rr6gnf7.oxygen-demo.qsandbox.me/?qcmd=auto_login&user=demo&pass=demo&next=admin.

I'm not able to notice any speed difference. Please can you let me know if it works correctly for you?

**Comment by SekSion** [comment]
On the sandbox i dont see a performance issue... Can you provide the pack you used ? Maybe j was using something wrong...

**Comment by KittenCodes** [comment]
Sure - here's the full symbol-defs.svg file: https://d.pr/f/QtGc88.
